packages/dataverse/docs/GET_STARTED.md

```diff
@@ -0,0 +1,512 @@
+```
+
+### `map()`
+
+It is also possible to create a derivation based on an existing derivation:
+
+```typescript
+const niceNumberB = new Box(5)
+const isNiceNumberEvenD = niceNumberB.derivation.map((v) => v % 2 === 0)
+
+// the following line will print '5, false' to the console
+console.log(niceNumberB.get(), isNiceNumberEvenD.getValue())
+```
+
+The new derivation will be always up to date with the value of the original
+derivation:
+
+```typescript
+import {Box} from '@theatre/dataverse'
+
+const niceNumberB = new Box(5)
+const isNiceNumberEvenD = niceNumberB.derivation.map((v) =>
+  v % 2 === 0 ? 'even' : 'odd',
+)
+
+const untap = isNiceNumberEvenD.changesWithoutValues().tap(() => {})
+
+const interval1 = setInterval(untap, 5000)
+const interval2 = setInterval(() => {
+  niceNumberB.set(niceNumberB.get() + 1)
+  console.log(
+    `${niceNumberB.get()} is an ${isNiceNumberEvenD.getValue()} number.`,
+  )
+}, 1000)
+
+// clear the intervals
+setTimeout(() => {
+  clearInterval(interval1)
+  console.log('interval1 is cleared.')
+}, 7000)
+
+setTimeout(() => {
+  clearInterval(interval2)
+  console.log('interval2 is cleared.')
+}, 7000)
+```
+
+### `prism()`
+
+At this point we can make derivations that track the value of an other
+derivation with [the `.map()` method](#map), but what if we want to track the
+value of multiple derivations at once for the new derivation? This is where the
+`prism()` function comes into play.
+
+Let's say that we have two derivations and we want to create a derivation that
+returns the product of their values. In the spreadsheet analogy it would be like
+having two cells with two functions and third cell that contains a function that
+calculates the product of the previous two cells. Whenever the first two cells
+recalculate their value, the third cell will also do the same.
+
+Here's how we would solve this problem in `dataverse`:
+
+```typescript
+import {Box, prism} from '@theatre/dataverse'
+
+const widthB = new Box(1)
+const heightB = new Box(2)
+const padding = 5
+
+const widthWithPaddingD = widthB.derivation.map((w) => w + padding)
+const heightWidthPaddingD = heightB.derivation.map((h) => h + padding)
+
+const areaD = prism(() => {
+  return widthWithPaddingD.getValue() * heightWidthPaddingD.getValue()
+})
+
+console.log('area: ', areaD.getValue())
+widthB.set(10)
+console.log('new area: ', areaD.getValue())
+```
+
+### `usePrism()`
+
+You can also use derivations inside of React components with the `usePrism()`
+hook from the `@theatre/react` package, which accepts a dependency array for the
+second argument. All the values that are tracked should be included in the
+dependency array.
+
+> Note that all the changes of the derivations' values inside the callback
+> function (first argument of `usePrism()`) will be tracked, since `usePrism()`
+> uses the `prism()` function under the hood. However, if the derivations are
+> not provided in the dependency array of `usePrism()`, then it will not know if
```

Comment:
I'd say "If the prism uses a value that is not a derivation (such as a simple number, or a pointer), then you need to provide that value to the dependency array.

Reply:
Okay, that does sound more clear. I'll change this.

Reply:
This is where the change got implemented: https://github.com/theatre-js/theatre/pull/116#discussion_r843774981